server/src/test/java/org/elasticsearch/index/mapper/BlockSourceReaderTests.java

```diff
@@ -90,8 +90,10 @@ private void loadBlock(MapperService mapperService, LeafReaderContext ctx, Consu
         var sourceLoaderLeaf = sourceLoader.leaf(ctx.reader(), null);
 
         assertThat(loader.rowStrideStoredFieldSpec().requiresSource(), equalTo(true));
+        var storedFieldSpec = loader.rowStrideStoredFieldSpec()
+            .merge(new StoredFieldsSpec(true, false, sourceLoader.requiredStoredFields()));
```

Comment:
Not sure what this is doing? `loader.rowStrideStoredFieldSpec()` already requires source via the above assertion, so isn't this a no-op?

Reply:
I'm not sure what its for as I copied it from somewhere else, but without this change, one of the tests does not pass. It might have something to do with ignored_source. Previously, an unintended consequence of changing the `store` default was that we were storing values in ignored_source.

Reply:
It is a testing issue. The required stored fields will otherwise not contain the `field` stored field.